static_src/components/users_org_user_selector.jsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Renders a form that allows org users to invite new users
+ * to cloud.gov
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Action from './action.jsx';
+import FormStore from '../stores/form_store';
+import { Form, FormSelect } from './form';
+import PanelDocumentation from './panel_documentation.jsx';
+import userActions from '../actions/user_actions';
+import { validateString } from '../util/validators';
+
+const AUDITOR_NAME = 'auditors';
+const SPACE_AUDITOR_NAME = 'space_auditor';
+const USERS_PARENT_ENTITY_USER_FORM_GUID = 'users-parent-entity-users-form';
+
+const propTypes = {
+  orgUsersSelectorDisabled: PropTypes.bool,
+  currentUserAccess: PropTypes.bool,
+  parentEntityUsers: PropTypes.array,
+  error: PropTypes.object,
+  parentEntity: PropTypes.string,
+  currentEntityGuid: PropTypes.string,
+  currentEntity: PropTypes.string
+};
+const defaultProps = {
+  orgUsersSelectorDisabled: false,
+  currentUserAccess: false,
+  error: {}
+};
+
+export default class OrgUsersSelector extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.validateString = validateString().bind(this);
+    this._onSubmitForm = this._onSubmitForm.bind(this);
+  }
+
+  componentDidMount() {
+    FormStore.create(USERS_PARENT_ENTITY_USER_FORM_GUID);
+  }
+
+  _onSubmitForm(errs, values) {
+    const { currentEntity } = this.props;
+    const { currentEntityGuid } = this.props;
+    const apiKey = AUDITOR_NAME;
```

Comment:
there's another refactor while we are talking about it, (but outside the scope of this PR)
addUserRoles currently only takes one role but it and the argument are plural `roles`